AuthenticatorShared/UI/Vault/ItemList/ItemList/ItemListEffect.swift

```diff
@@ -15,6 +15,12 @@ enum ItemListEffect: Equatable {
     ///
     case copyPressed(_ item: ItemListItem)
 
+    /// The Move to Bitwarden item button was pressed.
+    ///
+    /// - Parameter item: The item that should be moved.
+    ///
+    case moveToBWPressed(_ item: ItemListItem)
```

Comment:
🤔 I see other places where we're using `Bitwarden` as the whole word instead of the abbreviation `BW` like in the `ItemListState`. Could the whole word be maintained? In this case I don't think it's very long to abbreviate.

Reply:
👍 Good note. I've adjusted that to the full Bitwarden here and in the function names so that we're consistent. Updated in latest.